app/__init__.py

```diff
@@ -136,7 +142,8 @@ def required_authentication():
 
     @app.before_request
     def record_user_agent():
-        statsd_client.incr("user-agent.{}".format(process_user_agent(request.headers.get('User-Agent', None))))
+        statsd_client.incr(
```

Comment:
Spacing and line returns look off, think we should undo all of the 'linting' related changes once your editor is configured properly?